core/src/main/java/bisq/core/trade/protocol/tasks/ApplyFilter.java

```diff
@@ -60,8 +61,9 @@ protected void run() {
                 failed("Payment method is banned.\n" +
                         "Payment method=" + trade.getOffer().getPaymentMethod().getId());
             } else if (filterManager.isPeersPaymentAccountDataAreBanned(paymentAccountPayload, appliedPaymentAccountFilter)) {
+            	String paymentDetails = Res.getWithCol("payment.altcoin.sender.address", trade.getOffer().getCurrencyCode());
```

Comment:
This absolutely wrong it would change the paymentDetails displayed e.g. for MoneyBeam you would replace `Res.get(paymentMethodId) + " - " + Res.getWithCol("payment.account") + " " + accountId;` with `Res.getWithCol("payment.altcoin.sender.address", trade.getOffer().getCurrencyCode());`